docs/source/guide/api/swagger.json

```diff
@@ -505,7 +505,7 @@
                     "scope": {
                         "title": "Scope",
                         "type": "string",
-                        "description": "Add scopes separeted by space. Default user scopes. Users might not have all scopes. Available scopes: read:bootstrap read:settings read:targets read:tasks read:token write:bootstrap write:targets write:token delete:targets",
+                        "description": "Add scopes separeted by space. Default user scopes. Users might not have all scopes. Available scopes: read:bootstrap read:settings read:tasks read:token write:bootstrap write:targets write:token delete:targets",
```

Comment:
You don't need to change swagger.json.
It is done automatically when changed the code running `make docs`

It comes from here:
https://github.com/vmware/repository-service-tuf-api/blob/a41294fba0dd2a6c412f271f582b23e106570c8c/repository_service_tuf_api/token.py#L28-L33

Reply:
I used `make docs` locally to generate the changes again.